src/app/__init__.py

```diff
@@ -1,26 +1,27 @@
-import os
 
-from dotenv import load_dotenv
+from app import routes
 from flask import Flask, request
+from dotenv import load_dotenv
+
+from .models.timelinepost import TimelinePost
+import os
+
+
 from peewee import *
 
 load_dotenv()
 app = Flask(__name__)
-app.config["SECRET_KEY"] = os.getenv("APP_SECRET")
+# app.config["SECRET_KEY"] = os.getenv("APP_SECRET")
```

Comment:
At some point I did something with the app that required an app secret, but can't remember what was it. Why did you remove the secret?

Reply:
I will uncomment this out and resubmit 🙌